File: VICSApp/CoreSystem/DHCD/FormUI/Report/frmReport.cs
using pmDHCD.Report;
using System;

namespace pmDHCD
{
    public partial class frmReport
    {
        public frmReport()
        {
            InitializeComponent();
        }

        private phieubieuquyet1 cr = new phieubieuquyet1();

        public object Phieubieuquyet
        {
            get
            {
                return cr;
            }

            set
            {
                cr = (phieubieuquyet1)value;
            }
        }

        private void frmReport_Load(object sender, EventArgs e)
        {
            CrystalReportViewer1.ReportSource = cr;
            CrystalReportViewer1.Show();
        }
    }
}

File: VICSApp/CoreSystem/DHCD/FormUI/Report/frmReport.vb
Public Class frmReport

    Dim cr As New phieubieuquyet1
    Public Property Phieubieuquyet()
        Get
            Return cr
        End Get
        Set(ByVal value)
            cr = value
        End Set
    End Property

    Private Sub frmReport_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load

        CrystalReportViewer1.ReportSource = cr
        CrystalReportViewer1.Show()

    End Sub
End Class